octoprint_mrbeam/static/js/app/view-models/design-store.js

```diff
@@ -11,23 +11,31 @@ $(function () {
         // 'http://localhost:8080';
         self.DESIGN_STORE_IFRAME_HEALTHCHECK_SRC =
             self.DESIGN_STORE_IFRAME_SRC + "/api/healthcheck";
+        self.DESIGN_STORE_TAB_ELEMENT = $("#designstore_tab_btn");
```

Comment:
I like the idea of storing a reference to the tab button here, although this reference could be lost if for some reason the object is detached of the DOM right? Somehow I've seen this behaviour in our plugin before... I think it's safer to always look for the element like $('#element'). What we could do is save the reference to the button id and reuse it. What do you think @khaledsherkawi ?

Reply:
Oh, yea that's 100% the case and that is why I didn't do that in the material store as the material store tab is dynamically rendered. However, the design store tab is not so it is always attached to the DOM. If we change the behavior to match the one of the material store then this needs to be changed as well.